src/Shared/SkipLocalsInit.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Used to indicate to the compiler that the .locals init flag should not be set in method headers.
+#if NET5_0
+[module: System.Runtime.CompilerServices.SkipLocalsInit]
+#endif
```

Comment:
Add an else #error clause that says the TFM check needs to be updated. That's how we check for future TFMs.